content/applications/productivity/calendar/outlook.rst

```diff
@@ -115,23 +177,27 @@ Sync with Outlook
 Sync Odoo Calendar and Outlook
 ------------------------------
 
-In the Odoo database, go to the :guilabel:`Calendar` module and click the :guilabel:`Outlook` sync
-button. The page will redirect to a Microsoft login page, and the user is asked to log in to their
-account, if they are not already, and grant the required permissions.
+In the Odoo database, go to the Calendar module and click the Outlook sync button on the right side
+of the page.
 
 .. image:: outlook/outlook-sync-button.png
    :align: center
    :alt: The "Outlook" sync button in Odoo Calendar.
 
 The synchronization is a two-way process, meaning that events are reconciled in both accounts
-(Outlook and Odoo).
+(Outlook and Odoo). The page will redirect to a Microsoft login page, and the user is asked to log
+in to their account, if they are not already. Finally, grant the required permissions by clicking
+:guilabel:`Accept`.
+
+.. image:: outlook/accept-terms.png
+   :align: center
+   :alt: Authentication process on Microsoft Outlook OAuth page.
 
 .. note::
    All users that want to use the synchronization simply need to :ref:`sync their calendar with
    Outlook <outlook/sync>`. The configuration of Microsoft's Azure account is only done once, as
```

Comment:
```suggestion
   *Outlook* <outlook/sync>`. The configuration of Microsoft's *Azure* account is only done once, as
```

Reply:
I left out italics inside the reference, Sam said to shy away from stacking formating.